diamm/helpers/object_pagination.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict
+import re
+
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
+
+class Object_Pagination(PageNumberPagination):
```

Comment:
But you don't need the templates either. If you define the page size it should automatically paginate. Make sure you check the `?format=json` view of the page when you do paginate it so that you can see what the available properties are.